packages/target-chrome-core/src/create-chrome-target.js

```diff
@@ -176,7 +178,7 @@ function createChromeTarget(
       }
     };
 
-    client.captureScreenshot = withRetries(options.chromeRetries)(
+    client.captureScreenshot = withRetries(options.chromeRetries, 500)(
```

Comment:
We could give `500`  a nice name and move it to the top of the module

What do you think? 🤔

Reply:
Fixed!